frontend/pages/Homepage/cards/WelcomeHost/WelcomeHost.tsx

```diff
@@ -16,7 +16,7 @@ import Modal from "components/modals/Modal";
 import LaptopMac from "../../../../../assets/images/laptop-mac.png";
 import LinkArrow from "../../../../../assets/images/icon-arrow-right-vibrant-blue-10x18@2x.png";
 import IconDisabled from "../../../../../assets/images/icon-action-disable-red-16x16@2x.png";
-import IconPassed from "../../../../../assets/images/icon-check-circle-green-16x16@2x.png";
```

Comment:
Increase the image size of `icon-check-circle-green-16x16@2x.png` to x2 the display size, which would be a minimum of 32px x 32px. Keep the filename as-is (unless image dimensions change).